saleor/graphql/translations/tests/test_translations.py

```diff
@@ -2308,3 +2308,189 @@ def test_product_and_attribute_translation(user_api_client, product, channel_USD
+                        richText
+                        translation(languageCode: $languageCode) {
+                            name
+                            richText
+                            }
+                        }
+            }
+        }
+    }
+    """
+    variables = {
+        "id": variant_id,
+        "kind": TranslatableKinds.VARIANT.name,
+        "languageCode": LanguageCodeEnum.PL.name,
+    }
+
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_translations]
+    )
+    data = get_graphql_content(response)["data"]
+
+    translations_response = data["translation"]["attributeValues"][0]
+    assert translations_response["name"] == attribute_value.name
+    assert translations_response["richText"] == dummy_editorjs(
+        "Rich text attribute content.", json_format=True
+    )
+    assert translations_response["translation"]["richText"] == dummy_editorjs(
+        "Test_dummy_data", json_format=True
+    )
+
+
+def test_page_attribute_value_rich_text_translation(
+    staff_api_client,
+    page_with_rich_text_attribute,
+    permission_manage_translations,
+    page_type_with_rich_text_attribute,
+    permission_manage_pages,
+):
+    rich_text = dummy_editorjs("Test_dummy_data")
+    variant_attr = page_type_with_rich_text_attribute.page_attributes.first()
+    attribute_value = variant_attr.values.first()
+    attribute_value.translations.create(language_code="pl", rich_text=rich_text)
+
+    page_id = graphene.Node.to_global_id("Page", page_with_rich_text_attribute.id)
+
+    query = """
+    query translation(
+        $   kind: TranslatableKinds!, $id: ID!, $languageCode: LanguageCodeEnum!
+        ){
+            translation(kind: $kind, id: $id){
+                __typename
+                ...on PageTranslatableContent{
+                            attributeValues{
+                            name
+                            richText
+                            translation(languageCode: $languageCode) {
+                                name
+                                richText
+                                }
+                            }
+                        }
+                    }
+                }
+        """
+
+    variables = {
+        "id": page_id,
+        "kind": TranslatableKinds.PAGE.name,
+        "languageCode": LanguageCodeEnum.PL.name,
+    }
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_translations]
+    )
+    data = get_graphql_content(response)["data"]
+
+    attribute_value_response = data["translation"]["attributeValues"][0]
+    assert attribute_value_response["name"] == attribute_value.name
+    assert attribute_value_response["richText"] == dummy_editorjs(
+        "Rich text attribute content.", json_format=True
+    )
+    assert attribute_value_response["translation"]["richText"] == dummy_editorjs(
+        "Test_dummy_data", json_format=True
+    )
```

Comment:
Not a big issue, but it would be easier to read these tests if we used actual translations of the original strings. Here we have two random English strings and it's harder to guess what output to expect, just by reading the code.